package.json

```diff
@@ -40,9 +40,7 @@
     "node": ">=10"
   },
   "dependencies": {
-    "got-scraping": "^3.2.3",
-    "underscore": "^1.9.1",
-    "undici": "^4.5.1"
+    "underscore": "^1.9.1"
```

Comment:
would be good if we get rid of this too, just peeked at the usage and should be easy

Reply:
done